### PR TITLE
Show output from bazel run for easier debugging (#134)

### PR DESCRIPTION
**Pull request by inferno-chromium**

### CODE REVIEW

File: src/local/butler/common.py [hunk 1]
```diff
@@ -145,7 +145,10 @@ def _print(s):
     if print_output:
       print(s)
 
-  _print('Running: %s' % command)
+  print_string = 'Running: %s' % command
+  if cwd:
+    print_string += " (dir='%s')" % cwd
```
Comment (Dor1s) [inline src/local/butler/common.py]: super nit: maybe `(cwd='%s')` ? `dir` is a bit ambiguous

Comment (inferno-chromium) [reply src/local/butler/common.py]: Done!

**Comment by evverx** [comment]
@inferno-chromium @Dor1s thank you!

Not that it matters but I was wondering what "/gcbrun" is and having followed all the links I ended up at https://console.cloud.google.com/cloud-build/builds/88085705-09b3-4724-adfd-dc8200182f1e?project=472119376969 where I was informed that I don't have access to view builds and triggers. Would it be possible to make it publicly available (which I think would be useful when external contributors opens PRs)?

**Comment by Dor1s** [comment]
> Would it be possible to make it publicly available (which I think would be useful when external contributors open PRs)?

Not sure about making that public, as it can be abused. Ideally, contributors should run the tests locally, and we'll trigger the tests on the CI prior to merging a patch. @oliverchang may have a better perspective

**Comment by evverx** [comment]
Ideally, yes, contributors should run the tests and PR won't be merged if the CI is unhappy, but if I hypothetically broke something, I'd have to wait for someone else to take a look and tell me what went wrong which doesn't seem to be as convenient (and fast) as immediate feedback I'd receive if I had access to the logs. But I agree that currently this is a hypothetical issue.

**Pull request merged**